### PR TITLE
votca*: add v1.6_rc1

### PR DESCRIPTION
**Pull request by junghans**

### CODE REVIEW

File: var/spack/repos/builtin/packages/votca-csg-tutorials/package.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+       This package contains the VOTCA coarse-graining tutorials.
+    """
+    homepage = "http://www.votca.org"
+    url      = "https://github.com/votca/csg-tutorials/tarball/v1.4"
+    git      = "https://github.com/votca/csg-tutorials.git"
+
+    version('develop', branch='master')
+    version('1.6_rc1', sha256='87c4d945d2bdcb247e985cd407b0767c441f7810f1237ae65a63617f136e2ac9')
+    version('1.5.1',   sha256='e35cea92df0e7d05ca7b449c1b5d84d887a3a23c7796abe3b84e4d6feec7faca')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/votca-csg-tutorials/package.py]: Should probably mark this version as `preferred=True` so we don't default to a release candidate.

File: var/spack/repos/builtin/packages/votca-csg-tutorials/package.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+       routine work when doing systematic coarse-graining of various
+       systems. The core is written in C++.
+
+       This package contains the VOTCA coarse-graining tutorials.
+    """
+    homepage = "http://www.votca.org"
+    url      = "https://github.com/votca/csg-tutorials/tarball/v1.4"
+    git      = "https://github.com/votca/csg-tutorials.git"
+
+    version('develop', branch='master')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/votca-csg-tutorials/package.py]: You can call this version `master` now that #1983 has been merged.

File: var/spack/repos/builtin/packages/votca-csg-tutorials/package.py (hunk 1 above)
Comment (junghans) [reply var/spack/repos/builtin/packages/votca-csg-tutorials/package.py]: Good point, fixed!

File: var/spack/repos/builtin/packages/votca-csg-tutorials/package.py (hunk 2 above)
Comment (junghans) [reply var/spack/repos/builtin/packages/votca-csg-tutorials/package.py]: Meh, I want to keep it consistent with the other votca packages.

**Pull request merged**